File: Code coach problems/Medium/Python/Military_Time.py
res = ""
if a[1][0] == 'A':
	c = 0
	for i in range(len(a[0])):
		if a[0][i].isdigit():
			c*=10
			c+=ord(a[0][i]) - ord('0')
		else:
		if c<10:
			res += '0'
		res += str(c) +a [0][i]
		c = 0
	if c<10:
		res += '0' + str(c)
	else:
		res+=str(c)
	else:
		c = 0
		for i in range(len(a[0])):
			if a[0][i].isdigit():
				c*=10
				c+=ord(a[0][i]) - ord('0')
			else:
				c+=12
				res += str(c) +a[0][i]
				c = 0
		if c<10:
			res += '0' + str(c)
		else:
			res += str(c)
print(res)

File: Code coach problems/Medium/C++/Military_Time.cpp
#include <iostream>
#include <string>
using namespace std;
int main()
{
	string a, b;
	cin >> a >> b;
	if(b[0] == 'A')
	{
		int c = 0;
		for(int i = 0; i < a.size(); i++)
		{
			if(isdigit(a[i]))
			{
				c*=10;
				c+=a[i] - '0';
			}
			else
			{
				if(c < 10) 
					cout << 0;
				cout << c << a[i];
				c = 0;
			}
		}
		if(c < 10) 
			cout << 0 << c;
		else 
			cout << c;
	}
	else
	{
		int c = 0;
		for(int i = 0; i < a.size(); i++)
		{
			if(isdigit(a[i]))
			{
				c*=10;
				c+=a[i] - '0';
			}
			else
			{
				c+=12;
				cout << c << a[i];
				c = 0;
			}
		}
		if(c < 10) 
			cout << 0 << c;
		else 
			cout << c;
	}
}
